src/Symfony/Component/DependencyInjection/Tests/Fixtures/CheckTypeDeclarationsPass/ExpectsIntegerArgument.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
to be replaced by the licence header (same below)

Reply:
also in fixture file?
if so, we could run the Fixer to apply this for other fixture files

Reply:
the thing is we'd like the licence header in fixtures (because why not), but then, no need to enforce other rules I'd say (at least we'll get many false-positives if we do)

Reply:
Yeah, we definitely need to introduce skipping rules like ECS has, I like it a lot.

But you can also have a separate ruleset for fixtures and run Fixer twice, pointing to the config file.